packages/hooking/manager.ts

```diff
@@ -256,18 +271,40 @@ export function registerAfterHook(
 export async function hookBuiltInFunction(hook: Hook): Promise<void> {
 	const { default: module } = await import(hook.pkg);
 	const originalFn = module[hook.target];
+	hookFunction(module, hook, originalFn);
+}
+
+function hookGlobalFunction(hook: Hook, vmContext?: vm.Context): void {
+	const originalFn = vmContext
+		? vm.runInContext(hook.target, vmContext)
+		: vm.runInThisContext(hook.target);
+	const id = callSiteId(hookManager.hookIndex(hook), hook.target);
+	const context = vmContext ?? globalThis;
+
+	if (originalFn === undefined || originalFn === null) {
```

Comment:
Is a truthy check better? `if(originalFn) { hookFunction(context, hook, originalFn); }`